### PR TITLE
Add Quick Links to the Readme for Accessibility, Shared Profile Data & Auth, and Discovery and Searching Across Collective Instances

### PR DESCRIPTION
**Pull request by allella**
1. You have read the contribution guidelines: Yes
2. Your pull request uses a descriptive title: Yes
3. Your pull request targets the master branch of chapter: Yes
4. Closes: Nothing, but shouldn't conflict with #61 either. May conflict with the README changes in #66 

Simple links to issues which have momentum and good titles where we can send frequent questions on these topics.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,9 +30,12 @@ We will focus on building an open API first. Then developers can use the API to
 - [Feature Dev / Stories](https://github.com/freeCodeCamp/chapter/issues)
 - [API](https://github.com/freeCodeCamp/chapter/issues/17)
 - [Documentation](https://github.com/freeCodeCamp/chapter/issues/12)
+- [Shared Profile Data & Auth Across Collectives](https://github.com/freeCodeCamp/chapter/issues/40)
+- [Discovery and Searching Across Collective Instances](https://github.com/freeCodeCamp/chapter/issues/33)
 - [Mobile Apps](https://github.com/freeCodeCamp/chapter/issues/20)
 - [UX - Design Landing page](https://github.com/freeCodeCamp/chapter/issues/5)
 - [Internationalization / Translation](https://github.com/freeCodeCamp/chapter/issues/21)
+- [Accessibility[(https://github.com/freeCodeCamp/chapter/issues/65)
```
Comment (ScottBrenner) [inline README.md]:
```suggestion
- [Accessibility](https://github.com/freeCodeCamp/chapter/issues/65)
```

Comment (chrismgonzalez) [inline README.md]: Need to reverse the closing brace so it links correctly.

Comment (ScottBrenner) [reply README.md]: Jinx! 😂

Comment (allella) [reply README.md]: Thanks. Guess I should test.

Comment (allella) [reply README.md]: Thanks. Guess I should test.

**Comment by allella** [comment]
Looks like #66 removed the Quick Links so not sure if there's a conflict.

Obviously I think these are helpful in the Readme, perhaps more than other things on there, but if folks think it's a waste then I'll stop.

The issue queue is already 2 pages, so even if/when we get the issue names easy to read/find we'll still be fighting with people asking "where is XYZ conversation" if they have to search for anything for more than 5 seconds.

**Pull request merged**